### ISSUE TITLE
[F.Rq.] Enumerate submissions on game pages

### ISSUE THREAD

**Issue by YoshiRulz** [TASVideos/tasvideos#1686]
I assume this was only done for pubs since subs could far outnumber pubs, but in practice there's often only a few. A simple flat list would save a click.

**Comment by vadosnaprimer** [comment]
the list can be giant for some games.

**Comment by adelikat** [comment]
> the list can be giant for some games.

https://tasvideos.org/1G

**Comment by Masterjun3** [comment]
> I assume this was only done for pubs since subs could far outnumber pubs

For reference, this was actually only done for pubs because every additional information we have to show, is additional load for the db for that single page load.
So the reasoning that it "saves" a page load does not apply in general. Users not clicking the submission list far outnumber those who do.

**Comment by YoshiRulz** [comment]
> the list can be giant for some games.

Yes, it would need to be conditional: use a list if the number of subs (looks to be exposed as `Model.Game.SubmissionCount`) is under a threshold, else use the current code.

**Comment by vadosnaprimer** [comment]
What's the scenario where seeing submissions right away is critical?

**Comment by YoshiRulz** [comment]
You can't currently filter subs by game, but you can filter by system, which should be enough assuming the queue stays under control. This would just be a small quality-of-life improvement to "game-first" browsing.

**Comment by vadosnaprimer** [comment]
This is why I asked how important it is in real life. How often do you have to do the extra click to actually go and see the submission list for a given game? I would say it's rare enough to remain hidden behind a link at all times (no threshold), because overwhelming majority of users only watch publicatons. And even if you don't want to **watch** the publications, simply checking current branch list for maintenance is more frequent than checking submissions for anything.
